tools/requires_log_to_webtreemap.rb

```diff
@@ -0,0 +1,49 @@
+if ARGV.length != 2
+  puts "Usage: #{$0} /path/to/requires.log /path/to/webtreemap"
+  puts
+  puts "This tool uses webtreemap, which can be obtained from https://github.com/evmar/webtreemap"
+  exit 1
```

Comment:
https://github.com/ManageIQ/manageiq/blob/master/spec/coverage_helper.rb#L6

Maybe, add the tools to this list:  https://github.com/ManageIQ/manageiq-gems-pending/blob/master/spec/coverage_helper.rb#L4